app/controllers/physical_server_controller.rb

```diff
@@ -33,7 +33,7 @@ def show_list
   end
 
   def textual_group_list
-    []
+    [%i(properties relationships assets)]
```

Comment:
I believe the branch that originated this Pull Request is behind master (that already have these changes)